client/src/components/ItemList.jsx

```diff
@@ -73,9 +89,15 @@ const ItemList = ({ match }) => {
             </p>
           )}
         </div>
-        <LinkButton primary userId={userId}>
-          Add Another Item
-        </LinkButton>
+        {linkToken != null && callbacks != null && (
+          <LinkButton
+            linkToken={linkToken}
+            userId={user.id}
+            callbacks={callbacks}
```

Comment:
could you explain to me briefly why the token and the callbacks are constructed and kept track of separately? Seems like they are separate state variables. Could this just be a single `config` object to avoid handling them separately and then merging them together later?

Reply:
I have turned this into one config object and one state variable